packages/descendants/src/Descendants.stories.tsx

```diff
@@ -194,3 +203,63 @@ export const MultipleContexts = () => {
     </TestParent>
   );
 };
+
+const Parent = ({
+  children,
+  open,
+  ...rest
+}: ComponentProps<'div'> & { open: boolean }) => {
+  const elRef = useRef(null);
+  const parentRef = useRef(null);
+  const { descendants, dispatch, getDescendants } =
+    useInitDescendants<HTMLDivElement>();
+
+  const handleTransition = () => {
+    console.log(getDescendants().map(d => d.id));
+  };
```

Comment:
curious what we're looking for when logging this? I wonder if it might be more helpful if we logged more info beyond id in a `console.table()`

Reply:
This is just to debug, and verify that the ids have refreshed. I like the idea — we  _could_ use `console.table` but I'm not sure it's worth it at this point